src/utils/cli.js

```diff
@@ -6,7 +6,14 @@ const options = {
     requiresArg: true
   },
   json: { describe: 'Return the output in JSON format' },
-  csv: { describe: 'Return the output in CSV format' }
+  csv: { describe: 'Return the output in CSV format' },
+  count: {
+    default: 25,
+    describe: 'The number of records to return'
```

Comment:
Is there a maximum?

Reply:
Yep good pick-up, it's 500. Will add to the message.